ui/src/react-admin/modules/content-page/components/blocks/BlockSpotlight/BlockProjectSpotlight.wrapper.tsx

```diff
@@ -40,10 +41,11 @@ export const BlockProjectSpotlightWrapper: FunctionComponent<ProjectSpotlightWra
 			const promises = elements.map(
 				async (projectInfo: ProjectSpotlightProps): Promise<ContentPageInfo | null> => {
 					const projectPath = projectInfo?.project?.value;
-					if (projectPath && projectPath.toString && projectPath.toString()) {
+					if (projectPath?.toString()) {
```

Comment:
```suggestion
					if (projectPath?.toString?.()) {
```

Although I don't know if we care about that much type safety here